### DISCUSSION TITLE
Libreoffice Save dialog - missing Flash drive shortcut

### DISCUSSION THREAD

**Discussion by Leifdin** [lxqt/lxqt#2078] [Q&A] [answered]
Hello,
currently, we are using LXQt (as a part of Lubuntu OS) on our school student machines. Students (age 8 to 15) frequently need to save their documents to flash drives. However, LibreOffice save dialog has no shortcut to mounted drives. Is there a way to get it there, as searching for mounted drives in file system is rather counter-intuitive for young users who have little to no experience with the system.

Thank you for your answers. Kindy,
Pavol Polonec
![screen](https://user-images.githubusercontent.com/10058811/134313374-d67b1d97-6e62-4b19-8aa4-33eed6fe1841.png)

**Comment by tsujan** [comment]
That's the default Qt dialog, not LXQt dialog. Libreoffice's Qt5 integration doesn't use LXQt dialog.

**Comment by stefonarch** [comment, marked as answer]
It's an old issue, and true also for network shares https://bugreports.qt.io/browse/QTBUG-6875?focusedCommentId=125639
Probably the best workaround is creating a symlink by dragging their mounted flashdrive to the users home directory and select "create link".
At least it's only one click away then.

**Comment by Leifdin** [comment]
Thank you for your time and answers, I think that the idea with symlink is perfectly viable for me. Thank you very much.
